### PR TITLE
CNTK Function Approximation Tutorial

### PR DESCRIPTION
**Pull request by dustinandrews**
Hi,

I'm really enjoying the CNTK toolkit. After struggling a bit and seeing lots of others posting about struggles with basic function approximation in CNTK I wanted to share what I have learned. I've created a new notebook tutorial with a basic function approximation model, standardization of values and explanations. It's heavily based on the classification notebook it's meant to augment. The functions it approximates are trivial in order to highlight how to use CNTK in a very basic way.

### CODE REVIEW

**Comment by sayanpa** [comment]
Hi Dustin,
Thanks for your contribution. We are creating a framework that would make it easy to integrated notebooks / example like yours. I will get back to you with more specific instructions that will make the integration easier and also make sure the notebook is tested routinely against the latest bits.

Sayan

**Comment by sayanpa** [comment]
Hi Dustin,
I am looking into your tutorial on approximating continuous functions. I would like you to move this tutorial to the Manual directory. Name the file Manual_How_to_approximate_continuous_functions. Would need a test be added to Tests/EndToEndTests/CNTKv2Python/Manual directory.

Once the directory move is done, you may want to go through the other manuals and revise your tutorial to word according to how a manual should read. Our manuals have a very large reach and appreciate your effort.

Thanks,
Sayan

**Comment by sayanpa** [comment]
Hi @dustinandrews,
The team took a closer look into your contribution. The unique part of your tutorial is the fact that you use a squared loss (for regression) instead of a cross-entropy with softmax loss (for classification). We do have a regression tutorial CNTK 106A and B, that uses the squared loss target with time series data. 

Given we haven't heard back, and the fact that we have an example for regression though not a elementary regression, we would appreciate if you could host your tutorial in a fork that we will be happy to point CNTK audience to.

Closing this issue for now. Please feel free to reopen the issue if you want.
